app/build.gradle

```diff
@@ -5,12 +5,12 @@ plugins {
 }
 
 android {
-    compileSdk 33
+    compileSdk 34
 
     defaultConfig {
-        applicationId "dev.loveth.roomdemo"
+        applicationId "com.example.roomdemo"
```

Comment:
wtf was this shit?

Reply:
hahah I have no idea, I copied whole buildgradle file at the end, cuz I was frustrated and thought at one point that there is the problem...